ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -466,6 +466,8 @@ private static VerifiableProperties buildFrontendVProps(File trustStoreFile)
     properties.put("netty.server.request.buffer.watermark", "1");
     TestSSLUtils.addSSLProperties(properties, "", SSLFactory.Mode.SERVER, trustStoreFile, "frontend");
     properties.put("frontend.account.service.factory", "com.github.ambry.account.InMemAccountServiceFactory");
+    // add key for singleKeyManagementService
+    properties.put("kms.default.container.key", TestUtils.getRandomKey(32));
```

Comment:
Where is this property defined?

Reply:
okay - I see where it is defined, but it seems to be outside of any config classes. How is this exposed to the user?

Reply:
This is used only for SingleKeyManagementService and not applicable for any key management service and hence had to do it this way. Let me know if you have a better way.